app/routes/sign-in-oidc.js

```diff
@@ -1,20 +1,20 @@
 const Joi = require('joi')
 const { authConfig } = require('../config')
-const { AUTH_COOKIE_NAME, AUTH_REFRESH_COOKIE_NAME } = require('../constants/cookies')
-const { POST } = require('../constants/http-verbs')
-const { getAccessToken, getRedirectPath } = require('../auth')
-const { decodeState } = require('../auth/defra-id/decode-state')
+const { AUTH_COOKIE_NAME } = require('../constants/cookies')
+const { GET } = require('../constants/http-verbs')
+const { validateState, decodeState, validateInitialisationVector, clearCache, getAccessToken, getRedirectPath } = require('../auth')
+const { cacheRefreshToken } = require('../auth/cache-refresh-token')
 
 module.exports = {
-  method: POST,
+  method: GET,
```

Comment:
We need to change this to `GET` so that the cookies are submitted to the route when redirected from Defra Id.  Otherwise, we won't get the session id from the session cookie and therefore we will treat it as a new session.  That means the state and initialisation vector won't be retrieved from the cache.